docker-compose.yaml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:
+  kubearmor-init:
+    image: kubearmor/kubearmor-init:stable
+    volumes:
+      - /opt/kubearmor/BPF:/opt/kubearmor/BPF
+   
+
+  kubearmor:
+    image: kubearmor/kubearmor:latest
+    volumes:
+      - /opt/kubearmor/BPF:/opt/kubearmor/BPF
+      - /sys/fs/bpf:/sys/fs/bpf
+      - /sys/kernel/security:/sys/kernel/security
+      - /sys/kernel/debug:/sys/kernel/debug
+      - /var/run/containerd/containerd.sock:/var/run/containerd/containerd.sock
+      - /run/containerd:/run/containerd
+      - /var/lib/docker:/var/lib/docker
+    privileged: true
+    cap_add:
+      - SETUID
+      - SETGID
+      - SETPCAP
+      - SYS_ADMIN
+      - SYS_PTRACE
+      - MAC_ADMIN
+      - SYS_RESOURCE
+      - IPC_LOCK
+      - CAP_DAC_OVERRIDE
+      - CAP_DAC_READ_SEARCH
```

Comment:
Hey, I think the aim of explicitly specifying the capabilities is to remove the escalated `privileged: true` permission, right? 
If that's the case, then were you able to verify the kubearmor container running without the `priviledged` access?

Reply:
@yashvardhanmishra please update.